logbook-json/pom.xml

```diff
@@ -34,6 +34,13 @@
         <dependency>
             <groupId>com.jayway.jsonpath</groupId>
             <artifactId>json-path</artifactId>
+            <exclusions>
```

Comment:
Wouldn't it be better if we have this exclusion in the parent pom's dependency management, so that it applies to all modules and scopes?